scripts/apps/contacts/components/Form/ProfileDetail.tsx

```diff
@@ -193,6 +193,9 @@ export class ProfileDetail extends React.PureComponent<IProps, IState> {
         const isRequired = get(this.state, 'requiredField', false);
         const MSG_REQUIRED = gettext('This field is required.');
         const isAssignable = this.state.contactType && this.state.contactType.assignable;
+        const minFieldMessage = gettext('At least one of [email, phone, mobile, twitter, ' +
```

Comment:
Can you rephrase the error message so it doesn't include any fields or if you want to include field names, can you get them from the function doing the validation instead of hardcoding?